### PR TITLE
BC-3348 - Added functionality to display status events.

### PR DESCRIPTION
**Pull request by Michaellinaresxk**
Screen Shot 2023-05-02 at 7 12 10 PM
# Description
Create functionality to add color status priority in status alert component, in the same time add reponsive properties and remove text limit in the same component.

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-3348

## Changes
-Added shadows
-Title and text message no limit
-Responsive features and scroll bar
-Added shadow
-Added created_at datetime

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: static/scripts/helpers/AlertMessageController.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
+/* eslint-disable max-len */
```
Comment (VikDavydiuk) [inline static/scripts/helpers/AlertMessageController.js:1]: Can we just decrease the lengths of the needed lines? If not, we can disable the eslint rule just for spesific lines

File: static/scripts/helpers/AlertMessageController.js [hunk 2]
```diff
@@ -23,111 +23,49 @@ class AlertMessageController {
...
 			item.className = 'alert-item';
 			item.innerHTML = `
-			<div class="alert-date text-nowrap text-muted">
-					${datetime.fromNow(message.timestamp)}
-			</div>
 			<div class="alert-title">${icon} ${message.title}</div>
 			${message.text}
-			${url}
+			<br>
+			<div class="alert-date text-nowrap text-muted" style="float: left;">Updated: ${datetime.fromNow(message.timestamp)} <span style="margin-right: 2px;"></span>|</div>
```
Comment (VikDavydiuk) [inline static/scripts/helpers/AlertMessageController.js:33]: Avoid using float and magic number if  it's possible

File: static/scripts/helpers/AlertMessageController.js [hunk 3]
```diff
@@ -23,111 +23,49 @@ class AlertMessageController {
...
 			<div style="clear: both;"></div>`;
 		} else {
 			item.className = 'alert alert-info alert-card';
 			item.innerHTML = `<h6 style="overflow: hidden; text-overflow: ellipsis;">${icon} ${message.title}</h6>
-			<div class="text-muted" style="float: left;">${datetime.toDateTimeString(message.timestamp)}</div> <br>
-			${messageText}
-			${url}
+			${message.text}
+			<br>
+			<div class="text-muted" style="float: left;">Updated: ${datetime.toDateTimeString(message.timestamp)}<span style="margin-right: 2px;"></span>|</div>
```
Comment (VikDavydiuk) [inline static/scripts/helpers/AlertMessageController.js:41]: The same

File: static/scripts/helpers/AlertMessageController.js (hunk 2 above)
Comment (Michaellinaresxk) [reply static/scripts/helpers/AlertMessageController.js:33]: In this situation im working with the same code styles that is place it on legacy client in current file to keep the consistency.

File: static/scripts/helpers/AlertMessageController.js (hunk 3 above)
Comment (Michaellinaresxk) [reply static/scripts/helpers/AlertMessageController.js:41]: In this situation im working with the same code styles that is place it on legacy client in current file to keep the consistency.

File: static/scripts/helpers/AlertMessageController.js [hunk 4]
```diff
@@ -23,111 +22,55 @@ class AlertMessageController {
...
 			item.className = 'alert alert-info alert-card';
 			item.innerHTML = `<h6 style="overflow: hidden; text-overflow: ellipsis;">${icon} ${message.title}</h6>
-			<div class="text-muted" style="float: left;">${datetime.toDateTimeString(message.timestamp)}</div> <br>
-			${messageText}
-			${url}
+			${message.text}
+			<br>
+			<div class="text-muted" style="float: left;"> Updated: ${datetime.toDateTimeString(message.timestamp)}</div>
+			<div class="text-muted" style="float: left;">
+				<span>|</span> Created: ${datetime.toDateTimeString(message.created_at)}
```
Comment (muratmerdoglu-dp) [inline static/scripts/helpers/AlertMessageController.js:46]: i18n keys should be used for `Updated` and `Created` words.

**Review by muratmerdoglu-dp — CHANGES_REQUESTED** [review]

**Review by VikDavydiuk — APPROVED** [review]

**Review by muratmerdoglu-dp — APPROVED** [review]

**Pull request merged**